rand_gen/src/main.rs

```diff
@@ -0,0 +1,31 @@
+extern crate rand;
+use rand::Rng;
+use std::fs;
+
+
+fn main() {
+    
+    let mut count=0;
+    
+    let mut paths: Vec<_> = fs::read_dir("./").unwrap()
+    .map(|r| r.unwrap())
+    .collect();
+    paths.sort_by_key(|dir| dir.path());
+    
+    
+    for _i in paths.iter(){
+        count = count+1;   
+    }
+
+
+    let random_number= rand::thread_rng().gen_range(0,count);
```

Comment:
I tend to prefer avoiding indexing in rust. Try `let path = rand::thread_rng().choose(&paths)?;`